lib/src/settings.rs

```diff
@@ -135,6 +135,12 @@ impl UserSettings {
             .unwrap_or_else(|_| "push-".to_string())
     }
 
+    pub fn default_command(&self) -> String {
+        self.config
+            .get_string("ui.default-command")
+            .unwrap_or("log".to_string())
+    }
+
```

Comment:
I think we should inline this at the call site. Since this is in library code (under `lib/`), it shouldn't know about anything specific to the CLI.

Reply:
Good call...I hadn't thought about that boundary. I'll move it to the call site.